src/launcher/java/org/truffleruby/launcher/Launcher.java

```diff
@@ -89,6 +89,17 @@ public static void main(boolean isGraal, String[] args) throws Exception {
         System.exit(exitCode);
     }
 
+    private static String getEngineVersion() {
+        final String systemVersion = System.getProperty("org.graalvm.version");
+
+        // GraalVM dev builds set the version to "dev" and expect us to do something reasonable with it.
```

Comment:
They literally set it to "dev", not "0.32-dev"?

Reply:
Correct.